frontend/src/main/web/src/components/OverlayTrigger/readme.md

```diff
@@ -0,0 +1,2 @@
+Based on [React Bootstrap Overlay Trigger](https://github.com/react-bootstrap/react-bootstrap/blob/master/src/OverlayTrigger.js).
```

Comment:
Still doesn't actually say what this thing is. Is a concise description of this possible?